src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
```

Comment:
Improve your naming, if you create Lottery object so the variable should also be lottery.